src/bonds.cpp

```diff
@@ -570,6 +735,68 @@ TEST_CASE("[Faunus] BondData") {
         }
     }
 
+        SUBCASE("HarmonicDihedral") {
+        ParticleVector p_45deg(4, Particle());
```

Comment:
second argument `Particle()` is not needed

Reply:
Fixed with f1feccc.